onnxruntime/core/framework/session_state.cc

```diff
@@ -1410,6 +1410,23 @@ Status SessionState::FinalizeSessionStateImpl(const std::basic_string<PATH_CHAR_
 
 #endif
 
+#ifdef ORT_ENABLE_STREAM
+  // set the has_device_stream_enabled_ep_ flag
+  has_device_stream_enabled_ep_ = true;
```

Comment:
should this be initially set to false instead? it gets set to true again below.

Reply:
my bad. thanks for catching this.